.github/workflows/test.yml

```diff
@@ -7,15 +7,17 @@ jobs:
     strategy:
       matrix:
         java_version: [
-          "adopt@1.8.0-292"
+          8
         ]
     steps:
       - name: Checkout
         uses: actions/checkout@v3
       - name: Setup Scala
-        uses: olafurpg/setup-scala@v13
+        uses: actions/setup-java@v3
         with:
```

Comment:
Is this change needed for this PR? seems like GH does't like to run action changes in the same PR, so if possible, it would be good procedurally to have these in a separate one.

Reply:
OK. I will remove this change from this PR. And make this change another PR.